Source/WebCore/platform/graphics/GraphicsContext.cpp

```diff
@@ -81,6 +81,23 @@ void GraphicsContext::restore()
         m_stack.clear();
 }
 
+void GraphicsContext::addFilterEffectStyles(const Vector<FilterEffectStyle> styles)
+{
+    for (auto it = styles.rbegin(); it != styles.rend(); ++it) {
+        save();
+        setFilterEffectStyle(*it);
+        beginTransparencyLayer(1);
+    }
+}
+
+void GraphicsContext::removeFilterEffectStyles(unsigned count)
```

Comment:
That means there is no need for a "remove" function.

Reply:
Done. This function is moved to Filter::removeStyles().